ui/src/app/edge/live/Controller/Io/FixDigitalOutput/modal/modal.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { BUTTONS_FROM_CHANNEL_LINE, ONLY_NAME_LINE } from "src/app/edge/live/common/grid/modal/constants.spec";
```

Comment:
This should not import from grid; Move constants somewhere to common

Reply:
i agree